### PR TITLE
Branweb/python3 upgrade

### PR DESCRIPTION
**Pull request by branweb1**
Most changes boil down to:

- msgpack decodes to python3 bytes rather than unicode strings, so updated the application and tests to reflect that
- tornado moved away from decorators for async to async/await, so updated the app and the tests to reflect that
- readme and examples folder contained stale info dating from when this thing used json, not msgpack, so got rid of all that.

### CODE REVIEW

File: .TestDockerfile [hunk 1]
```diff
@@ -14,4 +13,4 @@ ADD dev-requirements.txt /app/
 RUN pip install -r /app/dev-requirements.txt
 ADD carto_renderer /app/carto_renderer
 
-RUN PYTHONPATH=/app py.test -vv /app/carto_renderer
+RUN PYTHONPATH=/app py.test -vv -W ignore::DeprecationWarning /app/carto_renderer
```
Comment (branweb1) [inline .TestDockerfile]: could remove this if it's a really bad idea. An test dependency was using a deprecated feature. The dep was updated a few days ago but not released yet: https://github.com/pytest-dev/pytest-asyncio/issues/141

**Comment by branweb1** [comment]
ah seems like the jenkins build needs fixing too. lame...ok working on it

**Comment by chitang** [comment]
lgtm, but I am not really a python programmer.  Not sure if you want to get a 2nd reviewer.

File: .TestDockerfile (hunk 1 above)
Comment (kaidamasaki) [reply .TestDockerfile]: I'd recommend waiting for the dep to update.

Comment (branweb1) [reply .TestDockerfile]: KIADA!!!

**Comment by branweb1** [comment]
retest this please

File: .TestDockerfile (hunk 1 above)
Comment (rozap) [reply .TestDockerfile]: hi Kaida!

**Comment by branweb1** [comment]
retest this please

**Comment by branweb1** [comment]
Ok removed "ignore warning" flag from the test-runner command. Going ahead with merge since I'd like this to back in staging for a while. Created this ticket to track updating pytest-asyncio to get rid of the test output warning: https://socrata.atlassian.net/browse/EN-39192

File: .TestDockerfile (hunk 1 above)
Comment (kaidamasaki) [reply .TestDockerfile]: Hi!!!

**Pull request merged**